suzieq/poller/worker/services/macs.py

```diff
@@ -52,6 +52,40 @@ def _add_mackey_protocol(self, entry):
                     else:
                         entry['mackey'] = '0'
 
+    def _clean_aos_data(self, processed_data, _):
+
+        entry_new = []
+
+        for entry in processed_data:
+
+            entry_dict = {}
+
+            protocol = entry.get('domain', -1)
+            vlan = entry.get('id', -1)
+            macaddr = entry.get('mac_address', -1)
+            #Where does this parameter go?
+            entry.get('type', -1)
```

Comment:
I believe this is a leftover, we don't have a `type` value field in the macs table. I think there was a discussion on slack about placing this piece information in the 'flags` field but I'm not sure whether that was cleared out.

Reply:
Adjusted to use the function, but gives the same results as before.

Reply:
Removed `type` field.